apps/gateway/package.json

```diff
@@ -30,7 +30,7 @@
     "hono": "^4.6.6",
     "ioredis": "^5.4.1",
     "lodash-es": "^4.17.21",
-    "promptl-ai": "^0.6.1",
+    "promptl-ai": "catalog:",
```

Comment:
What's a catalog? xD

Reply:
it allows you to define a single dependency versions for all packages using that dependency

Reply:
Oh nice